package.json

```diff
@@ -1,13 +1,24 @@
 {
   "name": "bcgov-digital-marketplace",
+  "jest": {
+    "verbose":true,
+    "moduleDirectories": [
+      "node_modules",
+      "src"
+  ],
+    "moduleNameMapper": {
```

Comment:
The rest of the repo uses shortened paths (see the `tsconfig` for details), so this config option sets up the same shortened paths. It would be better to get Jest to read the `tsconfig` so we don't have paths in two places

Reply:
I'd lean towards solving the root problem here, which is to ensure our jest environment is [using the same transform](https://github.com/bcgov/cas-ciip-portal/blame/d626a4113428ab61b4e52310b14ac6e2af146442/app/package.json#L180) as the rest of the codebase.

Reply:
oh, scratch that. Looks like it's [now the default](https://jestjs.io/docs/next/code-transformation#defaults) and I see you've got `babel-jest` in the package lower down.